app/controllers/webhook_deliveries_controller.rb

```diff
@@ -13,7 +13,7 @@ def index
       build_webhook_delivery(webhook_delivery_data)
     end
 
-    @total_pages = 1 # FIX ME: response.headers["X-Total-Count"]
+    @total_pages = response.headers["X-Total-Count"]
```

Comment:
Whoops sorry @Mitch-Henson I didnt see this pr before [merging this guy](https://github.com/artsy/doppler/commit/59663eafe4c6c74594cea96a7abab25a3eaa82ac)


Recently learned that `response.headers` in the line above is _not_ the same as the headers from Gravity's API response!! (Addressed that in that other PR)